api/src/main/java/io/kafbat/ui/util/GithubReleaseInfo.java

```diff
@@ -8,6 +8,7 @@
 
 @Slf4j
 public class GithubReleaseInfo {
+  public static final String GITHUB_RELEASE_INFO_ENABLED = "github.release.info.enabled";
```

Comment:
Please document this property in the docs too: https://github.com/kafbat/ui-docs

Reply:
Done: https://github.com/kafbat/ui-docs/pull/56

Also added the 2 pre-existing properties for this feature (timeout, update-rate)